src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIMetadataImportService.java

```diff
@@ -67,7 +66,7 @@ public UIMetadataImportService(MessageSource messageSource, ProjectService proje
 	 * @return {@link Map} of headers and rows from the csv or excel file for the user to select the header corresponding the
 	 * {@link Sample} identifier.
 	 */
-	public SampleMetadataStorage createProjectSampleMetadata(HttpSession session, long projectId, MultipartFile file) {
+	public SampleMetadataStorage createProjectSampleMetadata(HttpSession session, Long projectId, MultipartFile file) {
```

Comment:
At the bottom of this method there are 2 exception that get caught, but only log to the console.  This is old code of mine. I think we should return an error response to the suer so that they know what happened.